graphics/ZBar/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+name                    ZBar
+github.setup            mchehab zbar 0.23.90
+revision                0
+
+checksums               rmd160  ce5fb3bb7376ae0208cdb83baf706d447a11dfe5 \
+                        sha256  9581b5ab467f7b45ef0d7219e773c8dd0de07eb916f32f6828cd48b96ce50344 \
+                        size    1005439
+
+categories              graphics
+maintainers             {@knapoc knapoc} openmaintainer
```

Comment:
There's a similar issue with the gitup port. `knapoc` suggests your email is knapoc at macports dot org.

It should instead be `{@knapoc email.com:username} openmaintainer`, where the email is username at email dot com.